src/sentry/integrations/github/webhook.py

```diff
@@ -517,6 +520,7 @@ def handle(self, request: Request) -> HttpResponse:
         return HttpResponse(status=204)
 
 
+@region_silo_endpoint
```

Comment:
Some of GitHub's webhooks have to end up in control silo. The `installation.created` event is one that needs to stay in control.

Reply:
We actually don't react to an `installation.created` webhook, only [the `installation.deleted` one](https://github.com/getsentry/sentry/blob/master/src/sentry/integrations/github/webhook.py#L126) and in the parser it gets handled at the region silo with RPC methods.

Reply:
At the control silo, we would intercept the Github webhook requests with this parser: https://github.com/getsentry/sentry/blob/01592b6a976ddafe103a0d693b01995565ac6976/src/sentry/middleware/integrations/parsers/github.py

Reply:
My mistake. I conflated the github plugin and the integration. The plugin does use [installation.created webhook](https://github.com/getsentry/sentry/blob/31445be85fbd3ddd7c80dea12fa628795a52a6f6/src/sentry_plugins/github/webhooks/events/installation.py#L12-L21).